addon/styles/addon.scss

```diff
@@ -1,10 +1,10 @@
 @import 'node_modules/ember-frost-core/addon/styles/frost-theme';
 @import 'node_modules/ember-frost-core/addon/styles/frost-app';
-
 .frost-sort {
```

Comment:
wait..is frost-sort block actually being closed at line 32 in this file instead of at the end??